src/richie/apps/courses/admin.py

```diff
@@ -1,17 +1,30 @@
 """
```

Comment:
> Finally, it makes sure to respect the publication status of the cour in each language on the snapshot.

`s/cour/course/` in the commit message (`9db390fa4fd1b74b4d9d15bef436d75ec2b17cda`)